Kernel/Interrupts/PIC.h

```diff
@@ -33,18 +33,22 @@ namespace Kernel {
 class PIC final : public IRQController {
 public:
     PIC();
-    virtual void enable(u8 number) override;
-    virtual void disable(u8 number) override;
+    virtual void enable(GenericInterruptHandler&) override;
```

Comment:
A note for me so I'll fix it later - it's probably a better idea is to use `const GenericInterruptHandler&' signature instead.

Reply:
Fixed.